dot/network/notifications.go

```diff
@@ -256,13 +291,38 @@ func (s *Service) broadcastExcluding(info *notificationsProtocol, excluding peer
 				}
 			}
 
+			hsData, has = info.getHandshakeData(peer)
+			if !has {
+				continue
+			}
```

Comment:
shouldn't this be covered by the first case? `if hsData, has := info.getHandshakeData(peer); !has || !hsData.received`

Reply:
Done